Source/SwiftLintFramework/Models/Linter.swift

```diff
@@ -43,14 +44,35 @@ public struct Linter {
                     ruleTimes.append((id, -start.timeIntervalSinceNow))
                 }
             }
+
             return violations.filter { violation in
-                guard let violationRegion = regions
-                    .first(where: { $0.contains(violation.location) }) else {
-                        return true
+                guard let violationRegion = regions.first(where: { $0.contains(violation.location) }) else {
+                    return true
                 }
-                return violationRegion.isRuleEnabled(rule)
+                let enabled = violationRegion.isRuleEnabled(rule)
+                if !enabled {
+                    let identifiers = violationRegion.deprecatedAliasesDisablingRule(rule)
+                    if !identifiers.isEmpty {
+                        mutationQueue.sync {
```

Comment:
I've implemented that in d9eda65, but felt that creating a `struct` would be better than return tuples.